modules/node/src/transfer/transfer.repository.ts

```diff
@@ -32,6 +32,10 @@ export class LinkedTransferRepository extends Repository<LinkedTransfer> {
     });
   }
 
+  async findAllTransfers(available: boolean = true): Promise<LinkedTransfer[]> {
+    return await this.find({ where: { available }  });
+  }
```

Comment:
what is the `available` flag in the context of linked transfers?

Reply:
🤷‍♂️an unnecessary condition that appears not to have broken anything. I'll confirm it works without and if so remove it